src/outputs/outputs.cpp

```diff
@@ -157,78 +133,14 @@ Outputs::Outputs(Mesh *pm, ParameterInput *pin, SimTime *tm) {
-              << "Slice at x1=" << x1 << " in output block '" << op.block_name
-              << "' is out of range of Mesh" << std::endl;
-          PARTHENON_FAIL(msg);
-        }
-      }
-
-      if (pin->DoesParameterExist(op.block_name, "x2_slice")) {
-        Real x2 = pin->GetReal(op.block_name, "x2_slice");
-        if (x2 >= pm->mesh_size.x2min && x2 < pm->mesh_size.x2max) {
-          op.x2_slice = x2;
-          op.output_slicex2 = true;
-        } else {
-          msg << "### FATAL ERROR in Outputs constructor" << std::endl
-              << "Slice at x2=" << x2 << " in output block '" << op.block_name
-              << "' is out of range of Mesh" << std::endl;
-          PARTHENON_FAIL(msg);
-        }
-      }
-
-      if (pin->DoesParameterExist(op.block_name, "x3_slice")) {
-        Real x3 = pin->GetReal(op.block_name, "x3_slice");
-        if (x3 >= pm->mesh_size.x3min && x3 < pm->mesh_size.x3max) {
-          op.x3_slice = x3;
-          op.output_slicex3 = true;
-        } else {
-          msg << "### FATAL ERROR in Outputs constructor" << std::endl
-              << "Slice at x3=" << x3 << " in output block '" << op.block_name
-              << "' is out of range of Mesh" << std::endl;
-          PARTHENON_FAIL(msg);
-        }
-      }
-
-      // read sum options.  Check for conflicts with slicing.
-      op.output_sumx1 = pin->GetOrAddBoolean(op.block_name, "x1_sum", false);
-      if ((op.output_slicex1) && (op.output_sumx1)) {
-        msg << "### FATAL ERROR in Outputs constructor" << std::endl
-            << "Cannot request both slice and sum along x1-direction"
-            << " in output block '" << op.block_name << "'" << std::endl;
-        PARTHENON_FAIL(msg);
-      }
-      op.output_sumx2 = pin->GetOrAddBoolean(op.block_name, "x2_sum", false);
-      if ((op.output_slicex2) && (op.output_sumx2)) {
-        msg << "### FATAL ERROR in Outputs constructor" << std::endl
-            << "Cannot request both slice and sum along x2-direction"
-            << " in output block '" << op.block_name << "'" << std::endl;
-        PARTHENON_FAIL(msg);
-      }
-      op.output_sumx3 = pin->GetOrAddBoolean(op.block_name, "x3_sum", false);
-      if ((op.output_slicex3) && (op.output_sumx3)) {
-        msg << "### FATAL ERROR in Outputs constructor" << std::endl
-            << "Cannot request both slice and sum along x3-direction"
-            << " in output block '" << op.block_name << "'" << std::endl;
-        PARTHENON_FAIL(msg);
-      }
-
       // read ghost cell option
       op.include_ghost_zones = pin->GetOrAddBoolean(op.block_name, "ghost_zones", false);
 
       // read cartesian mapping option
       op.cartesian_vector = false;
 
       // read single precision output option
-      const bool is_hdf5_output =
-          (op.file_type == "rst") || (op.file_type == "ath5") || (op.file_type == "hdf5");
+      const bool is_hdf5_output = (op.file_type == "rst") || (op.file_type == "hdf5");
```

Comment:
`ath5` doesn't exist anywhere else in the code. We should either transform it to `hdf5` or just remove it.

Reply:
I'm in favor of removing. I've never seen it used anywhere.